packages/neon_framework/lib/src/storage/request_cache.dart

```diff
@@ -76,6 +77,10 @@ final class DefaultRequestCache implements RequestCache {
             await txn.execute('DROP TABLE cache');
             await onCreate(txn);
           }
+
+          if (oldVersion == 3) {
```

Comment:
```suggestion
          if (oldVersion <= 3) {
```
I know that if the old version was 2 or less then the DROP will be executed so the DELETE isn't necessary, but I think this is easier to read especially with future migrations also having to use <= and not ==